package.json

```diff
@@ -22,7 +22,7 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "concurrently \"nodemon server/server.js\" \"react-scripts start\"",
+    "start": "concurrently \"nodemon server/server.js --ignore client\" \"react-scripts start\"",
```

Comment:
Interesting addition, what does this do?

Reply:
Found it suggested online as a possible solution, I believe it is intended to ignore the dev server that CRA throws up and instead ensure all requests go straight to the node server